### PR TITLE
Issues/#83 rename to equibit

### PR DESCRIPTION
**Pull request by r-soltani**
As part of rebranding Bitcoin to Equibit (epic #60), the texts in the source codes that contain Bitcoin must change to Equibit. 
This PR also contains a Visual Studio Snippet to make it easier to add #ifdef preprocessors around code changes.

### CODE REVIEW

**Review by ealefad — APPROVED** [review]
Fails with only one error related to functional tests on Ubuntu which is predictable since functional test framework in /test/functional/ looks for 'bitcoind' executable. Windows doesn't run the functional tests so it's not failing in travis.

**Review by macterra — CHANGES_REQUESTED** [review]
Merge from dev to make the checks pass before merging

File: src/rpc/mining.cpp [hunk 1]
```diff
@@ -442,11 +446,18 @@ UniValue getblocktemplate(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_CLIENT_P2P_DISABLED, "Error: Peer-to-peer functionality missing or disabled");
 
     if (g_connman->GetNodeCount(CConnman::CONNECTIONS_ALL) == 0)
-        throw JSONRPCError(RPC_CLIENT_NOT_CONNECTED, "Bitcoin is not connected!");
```
Comment (KiarashN) [inline src/rpc/mining.cpp]: Make sure we don't change spaces to create "delete line"

File: src/init.cpp [hunk 2]
```diff
@@ -1229,11 +1229,19 @@ bool AppInitMain()
 
     // Warn about relative -datadir path.
     if (gArgs.IsArgSet("-datadir") && !fs::path(gArgs.GetArg("-datadir", "")).is_absolute()) {
+#ifdef BUILD_BTC
         LogPrintf("Warning: relative datadir option '%s' specified, which will be interpreted relative to the "
                   "current working directory '%s'. This is fragile, because if bitcoin is started in the future "
                   "from a different location, it will be unable to locate the current data files. There could "
                   "also be data loss if bitcoin is started while in a temporary directory.\n",
-            gArgs.GetArg("-datadir", ""), fs::current_path().string());
```
Comment (KiarashN) [inline src/init.cpp]: Please make sure the spaces stays the same so it doesn't delete the line

Comment (r-soltani) [reply src/init.cpp]: fixed

File: src/rpc/mining.cpp (hunk 1 above)
Comment (r-soltani) [reply src/rpc/mining.cpp]: fixed

File: build_msvc/bitcoin_config.h [hunk 3]
```diff
@@ -343,10 +343,18 @@
 #define PACKAGE_BUGREPORT "https://github.com/bitcoin/bitcoin/issues"
```
Comment (macterra) [inline build_msvc/bitcoin_config.h:345]: Add copyright notice? (here and all C++ files touched in this PR)

File: eqbtest/functional/rpc_users.py [hunk 4]
```diff
@@ -17,15 +17,15 @@ def set_test_params(self):
 
     def setup_chain(self):
         super().setup_chain()
-        #Append rpcauth to bitcoin.conf before initialization
+        #Append rpcauth to equibit.conf before initialization
         rpcauth = "rpcauth=rt:93648e835a54c573682c2eb19f882535$7681e9c5b74bdd85e78166031d2058e1069b3ed7ed967c93fc63abba06f31144"
         rpcauth2 = "rpcauth=rt2:f8607b1a88861fac29dfccf9b52ff9f$ff36a0c23c8c62b4846112e50fa888416e94c17bfd4c42f88fd8f55ec6a3137e"
         rpcuser = "rpcuser=rpcuser💻"
         rpcpassword = "rpcpassword=rpcpassword🔑"
```
Comment (macterra) [inline eqbtest/functional/rpc_users.py:25]: Is there really a key icon in the password?

**Review by macterra — CHANGES_REQUESTED** [review]
Add copyright notices now or defer work with a new ticket?

File: eqbtest/functional/rpc_users.py (hunk 4 above)
Comment (ealefad) [reply eqbtest/functional/rpc_users.py:25]: yes, it looks like there's a unicode sequence for a key emoji, UTF8="F0 9F 94 91"
http://www.endmemo.com/unicode/symbols_and_pictographs.php

Comment (macterra) [reply eqbtest/functional/rpc_users.py:25]: cool!

File: eqbtest/functional/feature_config_args.py [hunk 5]
```diff
@@ -28,7 +28,7 @@ def run_test(self):
         self.assert_start_raises_init_error(0, ['-datadir='+new_data_dir], 'Error: Specified data directory "' + new_data_dir + '" does not exist.')
```
Comment (r-soltani) [inline eqbtest/functional/feature_config_args.py:29]: @ealefad can you add the copyright statement to your python scripts

**Review by macterra — APPROVED** [review]

**Pull request merged**